helm-charts/charts/kube-starrocks/templates/init-pwd/job.yaml

```diff
@@ -0,0 +1,45 @@
+
+{{- if .Values.initPassword.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "kube-starrocks.name" . }}-credential
+  namespace: {{ .Release.Namespace }}
+stringData:
+  password: {{ .Values.initPassword.password }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: starrocks-initpwd
+  namespace: {{ .Release.Namespace }}
+spec:
+  template:
+    spec:
+      containers:
+      - name: {{ template "kube-starrocks.name" . }}-initpwd
+        image: {{ .Values.starrocksFESpec.image.repository }}:{{ .Values.starrocksFESpec.image.tag }}
+        imagePullPolicy: Always
+        command:
+        - /bin/bash
+        args:
+        - /opt/starrocks/fe_initpwd.sh
+        - {{ template "kube-starrocks.name" . }}-fe-0.{{ template "kube-starrocks.name" . }}-fe-search
+        - "9030"
+        - "SET PASSWORD = PASSWORD('{{ .Values.initPassword.password }}');"
```

Comment:
this will make the password plaintext in job yaml file, can we fill this as env vars and deduct it from the secret. in the fe_initpwd.sh read the env var directly.